flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -378,10 +378,12 @@ class SentryFlutterOptions extends SentryOptions {
   // ignore: invalid_use_of_internal_member
   set automatedTestMode(bool value) => super.automatedTestMode = value;
 
-  /// Indicates if SentryNavigatorObserver has been registered
-  /// This is used internally to manage web session tracking features
+  /// If app lifecycle trace generation is enabled, this sets the duration the app must
+  /// be in the background before a new trace id is generated starts upon resuming.
```

Comment:
Grammar: `...a new trace id is generated starts upon resuming.` -> `...a new trace id is generated upon resuming.`